product_docs/docs/transporter/ba_preview/getting_started/installing/index.mdx

```diff
@@ -6,32 +6,28 @@ navigation:
   - linux_x86_64
 ---
 
-To install EDB Transporter for internal purposes:
+Select a link to access the applicable installation instructions.
 
-### Accessing CloudSmith
+## Linux [x86-64 (amd64)](linux_x86_64)
 
-Log in to Okta, and navigate to the CloudSmith tile to access the dev CloudSmith repository. 
+### Red Hat Enterprise Linux (RHEL) and derivatives
 
-### Finding the appropriate package
+- [RHEL 8](linux_x86_64/transporter_rhel_8), [RHEL 7](linux_x86_64/transporter_rhel_7)
 
-Look for the last `cdcreader` version for the architecture and operating system you are using.
+- [Oracle Linux (OL) 8](linux_x86_64/transporter_rhel_8), [Oracle Linux (OL) 7](linux_x86_64/ransporter_rhel_7)
```

Comment:
Typo 
```suggestion
- [Oracle Linux (OL) 8](linux_x86_64/transporter_rhel_8), [Oracle Linux (OL) 7](linux_x86_64/transporter_rhel_7/)
```

Reply:
Also, in the EPAS docs, we also have Oracle Linux 9 documented. Just pointing this out here :).